Software.py

```diff
@@ -180,11 +180,14 @@ def check_and_send_unfocus_event(view):
     global editor_focused
     global last_focus_event_sent
 
-    if(editor_focused is False and last_focus_event_sent is not 'unfocus'):
-        # this will send off codetime events
-        PluginData.send_all_datas()
-        track_editor_action(**editor_action_params(view, 'editor', 'unfocus'))
-        last_focus_event_sent = 'unfocus'
+    if (editor_focused is False):
+        # set the 'isFocused' value within the SoftwareWallClock to False
+        blurWindow()
```

Comment:
Is this the magic that makes the clock stop?

Reply:
yes. it sets the "isFocused" variable to True and the timer uses that condition to determine whether it should add seconds.